dev-python/types-psutil/types-psutil-5.9.5.16.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Typing stubs for psutil"
+HOMEPAGE="https://pypi.org/project/types-psutil"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
```

Comment:
While the main policy of keywording for new packages is to only do on arches you can test (`~amd64`), I think in this case it is fine.

Why? because those are just types packages, no code running or such. Equivalent to installing assets. As an Arch Tester, what can I test here - that it installs correctly? This is why I think straight up keywording for all needed arches is fine.

Reply:
Yup. Also the main reason I added them was because trying to `pkgdev commit` mypy yelled at me that mypy was missing those dependencies on some arches -- so I figured it was best to add them myself rather than wait for someone else to say "well it's just a data package anyway". :D